### ISSUE TITLE
Binary file RDPharm3D.chm without source(?) in repo

### ISSUE THREAD

**Issue by mbanck** [rdkit/rdkit#2211]
**Description:**

Debian currently has to repack RDKit sources due to the `Docs/Programs/RDPharm3D/RDPharm3D.chm` file which is a binary blob (I think?) without source and/or a way to recreate it.

In case that `.chm` file is no longer relevant or needed, maybe it can be purged from the repository, or  (if it is needed) generated during the build?

To be clear, it is not a big deal for us to repack RDKit, so if it problematic for you to remove the file, nevermind.

- RDKit Version: 2018.9
- Platform: Debian

**Comment by bp-kelley** [comment]
.chm is a windows help file, it is actually html based but compiled.  Opening it in windows will probably get the license, I can try that when I get home.

----
Brian Kelley

**Comment by mbanck** [comment]
I guess it is about "preferred form of modification", which would be some HTML source, but I am not sure how easy it is to build/bundle `.chm` files yourself?

**Comment by bp-kelley** [comment]
Actually, there is zero reason to ship this on Linux I think, I can make a pr 

----
Brian Kelley

**Comment by mbanck** [comment]
To be clear, the problem is the presence of the `.chm` in the repo/release tarballs (we strip those out for Debian and repackage the tarball).

But to also be clear: I don't want to force you to take those down, if there is good reason for them to stick around, we will just continue to repackage, no problem

**Comment by bp-kelley** [comment]
Exactly.  Since they don’t actually work on Linux, I can remove them from the build.

----
Brian Kelley

**Comment by greglandrum** [comment]
This is ancient documentation for bits of the code that no longer even exist. I'll remove them.

Thanks for pointing it out @mbanck

**Comment by greglandrum** [comment]
FWIW: the sources to generate this file were all in the same directory. It's all removed after #2213 though

**Comment by greglandrum** [comment]
Resolved in #2213
